Conceptpower+Spring/src/integration-test/resources/output/conceptEntryNotFound.xml

```diff
@@ -0,0 +1 @@
+<conceptpowerReply xmlns:digitalHPS="http://www.digitalhps.org/"></conceptpowerReply>
```

Comment:
Currently master branch does not throw any error when concept entry is not found. This has been fixed in develop branch. Develop branch will state "No concept found" error message in output.